packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -44,4 +42,9 @@ typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q
 ///
 /// The `debugLabel` argument can be specified to provide a name to add to the
 /// [Timeline]. This is useful when profiling an application.
+typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q message, { String? debugLabel });
+
+/// A function that spawns an isolate and runs a callback on that isolate.
+///
+/// See [ComputeImpl] for usage details.
```

Comment:
Here and below, we have a` See also:` format we usually follow 
```suggestion
/// See also:
///
///   * [ComputeImpl], <brief description of what it is or how it relates>.
```

Reply:
Done.